lib/tests/streamlit/Server_test.py

```diff
@@ -246,6 +284,31 @@ def send_data_msg():
             finish_report(True)
             self.assertFalse(is_data_msg_cached())
 
+    @staticmethod
+    def _create_mock_report_session(*args, **kwargs):
+        """Create a mock ReportSession. Each mocked instance will have
+        its own unique ID."""
+        mock_id = mock.PropertyMock(
+            return_value="mock_id:%s" % ServerTest._next_report_id
+        )
+        ServerTest._next_report_id += 1
+
+        mock_session = mock.MagicMock(ReportSession, autospec=True, *args, **kwargs)
+        type(mock_session).id = mock_id
+        return mock_session
+
+    def _patch_report_session(self):
+        """Mock the Server's ReportSession import. We don't want
+        actual sessions to be instantiated, or scripts to be run.
+        """
```

Comment:
At some point we said to use plain `#` for descriptions of private methods.

Reply:
Whoa, really? I strongly disagree with, and am confused by, that. For example, my IDE auto-parses docstrings to provide inline context. 

What do we gain by having two different methods for documenting methods?

Reply:
I think """ """ are picked up by documentation tools.

I don't have a strong opinion and I won't block this PR for this. @tvst?